waku/v2/node/config.nim

```diff
@@ -390,6 +390,17 @@ type
       defaultValue: 1
       name: "discv5-bits-per-hop" .}: int
 
+    ## waku peer exchange config
+    wakuPeerExchange* {.
+      desc: "Enable waku peer exchange protocol (responder side): true|false",
+      defaultValue: false
+      name: "waku-px" }: bool
+
+    pxNode* {.
```

Comment:
Minor comment, but any way we can have these config items have a similar prefix? There is indeed a future refactor task to simplify and properly group config items, but at some point we started prefixing similar config items to achieve a little more order. For example we have `store`, `storeNode`, `store*`, `filter`, `filterNode`, etc. I imagine `wakuPeerExchange` is to differentiate it from `relayPeerExchange`, but perhaps the latter is already clear enough and we can just use `peerExchange` and `peerExchangeNode`?

Reply:
Indeed. And there is an implied "Waku..." prefix to all our protocols (WakuStore, WakuFilter, etc.). Thanks!